guides/k8s-microservices/src/main/paradox/index.md

```diff
@@ -0,0 +1,292 @@
+
+Chirper is configured to use nginx as the ingress controller.
+
+### Installation script
+
+_Below, the install script in the Chirper repository is dissected. You can find it in its
+entirety at `deploy/k8s/minikube/scripts/install`_
+
+Now that all the resources required for deployment have been described, this guide will cover how to automate the process
+of deploying them to Kubernetes.
+
+Deploying Chirper requires the following actions:
+
+* Start up your Minikube
+* Deploy Cassandra
+* Build Chirper Docker images
+* Deploy Chirper
+* Deploy nginx
+* Open the service in your browser
+
+Let's take a look at how these tasks can be scripted by analyzing `install`. Feel free to run these commands
+from your own terminal as well. Should you do that, make sure you've `cd`'d into the Chirper repository.
+
+#### Starting up your Minikube
+
+For simplicity's sake, it's nice to start from a fresh Minikube installation. The following resets the state of the local
+Minikube and configures the environment variables of your terminal session to point Docker to it.
+ 
+```bash
+(minikube delete || true) &>/dev/null && \
+minikube start --memory 8192 && \
+eval $(minikube docker-env)
+```
+
+#### Deploy Cassandra
+
+To deploy Cassandra to Kubernetes, the requisite resources must be created. The command below will create the resources, wait for
+Cassandra to start up, and show you its status.
+
+
+```bash
+kubectl create -f deploy/k8s/minikube/cassandra && \
+deploy/k8s/minikube/scripts/kubectl-wait-for-pods && \
+kubectl exec cassandra-0 -- nodetool status
+```
+
+```
+service "cassandra" created
+statefulset "cassandra" created
+Datacenter: DC1-K8Demo
+======================
+Status=Up/Down
+|/ State=Normal/Leaving/Joining/Moving
+--  Address     Load       Tokens       Owns (effective)  Host ID                               Rack
+UN  172.17.0.4  99.45 KiB  32           100.0%            9f5ffc06-ba53-4f7d-8fbb-c4a522ae3ef8  Rack1-K8Demo
+```
+
+_Refer to the files in the Chirper repository at `deploy/k8s/minikube/cassandra` for more details._
+
+#### Build Chirper Docker images
+
+Applications must be packaged as Docker images to be deployed to Kubernetes. By using 
+[fabric8's docker-maven-plugin](https://dmp.fabric8.io/), these images will be built and published to the Minikube
+repository. The command below will build Chirper and the Docker images using Maven and this plugin.
+
+```bash
+mvn clean package docker:build
+```
+
+```
+...
+
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time: 01:18 min
+[INFO] Finished at: 2017-07-13T11:45:25-05:00
+[INFO] Final Memory: 79M/460M
+[INFO] ------------------------------------------------------------------------
+```
+
+Next, inspect the images that are available. Note that the various Chirper services all have their own image. These will
+be deployed to the cluster.
+```
+docker images
+```
+
+```bash
```

Comment:
And I think we should remove `bash` from this code snippet since it's the output of the command.